### DISCUSSION TITLE
How i can avoid truncated descriptions?

### DISCUSSION THREAD

**Discussion by javiabellan** [scalar/scalar#3074] [Q&A]
If i write a very large markdown description (for example at info level) the end of content is hidden and trunctated with `…` character.

How i can avoid this?

**Comment by antlio** [comment]
hey @javiabellan, it supposed to be the case only when a section is collapsed but there might be a miss here. is there any more infos you can share here or a part of a specification?

**Comment by hwkr** [comment]
> If i write a very large markdown description (for example at info level) the end of content is hidden and trunctated with `…` character.
> 
> 
> 
> How i can avoid this?

Hi @javiabellan! Do you have a screenshot of the issue and an example specification file?

**Reply by antlio** [reply]
i just reproduced it here: https://sandbox.scalar.com/p/r31UP#tag/planets/POST/planets it should only be the case at the section level i'd say? 🙏

**Comment by javiabellan** [comment]
Sorry for the late reply. I just solved the problem by "hacking" your CSS: i made `.collapsed` class equal to `{}`
of the CDN file: "https://cdn.jsdelivr.net/npm/@scalar/api-reference"

**Reply by antlio** [reply]
hey @javiabellan, this should be resolved by #3077!
